app/src/androidTest/java/com/github/se/gatherspot/firebase/EventFirebaseConnectionTest.kt

```diff
@@ -245,7 +245,7 @@ class EventFirebaseConnectionTest {
 
   @Test
   fun fetchMyEventsWorks() =
-      runTest(timeout = Duration.parse("20s")) {
+      runTest(timeout = Duration.parse("60s")) {
         testLogin()
         Thread.sleep(10000)
```

Comment:
This should be replaced with : runBlocking { delay(1000) } 
Thread.sleep stops the logging in process so it's useless in waiting for login to be done. In addition it may even cause the login process to fail since the code needs to create a dialog with firebase.